unittests/optionaltype_pointer/testpointerapi.cpp

```diff
@@ -39,4 +40,20 @@ void TestPointerApi::test()
     QCOMPARE(*resp.out(), newVal);
 }
 
+void TestPointerApi::testOptionalArray()
+{
+    TNS__TestOperationResponse1 resp;
+    QCOMPARE(resp.outOptionalArray(), (QList<qint64> *)0);
```

Comment:
QVERIFY(!resp.outOptionalArray()) would be enough (and would avoid the C cast, to which I'm allergic due to bad experiences ;) )